src/RageSoundReader_Chain.cpp

```diff
@@ -25,13 +26,14 @@
  */
 RageSoundReader_Chain::RageSoundReader_Chain()
 {
-	m_iPreferredSampleRate = PREFSMAN->m_iSoundPreferredSampleRate;
-	if (m_iPreferredSampleRate == 0)
-	{
-		m_iPreferredSampleRate = kFallbackSampleRate;
-	}
-	
+	// The preferred sample rate for resampling when sounds have different rates.
+	// Initialized to 44100 Hz, assuming most audio libraries use CD-standard rates.

```

Comment:
Don't mention the value here as it may change and this comment will be stale. Move that to to definition of the valie